### ISSUE TITLE
Upcoming events widget on chapter pages should have link to all upcoming events

### ISSUE THREAD

**Issue by xMartin** [OpenTechSchool/www.opentechschool.org#222]
Currently the Upcoming Events widget provides links to a limited number of upcoming events on meetup.com and has a link to a calendar feed.

There's no hint that there might be even more upcoming events. Could be solved by a "more" link. Or we just add a link "all events on meetup.com" or something like that.

**Comment by gsambrotta** [comment]
I think your ""all events on meetup.com" is what currently "» Find Closest To Me" does. 
The idea is that you go to your chapter page and check the upcoming events. 
Also because on HP there are upcoming events from all OTS in the world so if you want to link to meetup, it will be a link to general OTS meetup page. Not sure is the best

**Comment by xMartin** [comment]
You are right. At least for the home page. I was referring to the chapter pages.
